src/sidebar/store/modules/test/annotations-test.js

```diff
@@ -527,4 +527,19 @@ describe('sidebar/store/modules/annotations', () => {
       assert.isTrue(store.annotationExists(annot.id));
     });
   });
+
+  describe('isAnnotationHighlighted', () => {
+    it('returns true if the provided annotation ID is in the set of highlighted annotations', () => {
+      const store = createTestStore();
+      store.highlightAnnotations(['1', '2']);
+      assert.isTrue(store.isAnnotationHighlighted({ id: '1' }));
+    });
+
+    [undefined, {}, { id: '2' }].forEach(annotation => {
+      it('returns false if the provided annotation ID is not in the set of highlighted annotations', () => {
+        const store = createTestStore();
+        assert.isFalse(store.isAnnotationHighlighted(annotation));
+      });
```

Comment:
I would combine these tests into one and make them depend more on the implementation doing what it is supposed to. As it currently stands, an implementation of `store.highlightAnnotations`/`store.isAnnotationHighlighted` like this would still pass:

```js
highlightAnnotations(ids) {
  return { highlighted: true }
}

isAnnotationHighlighted(state, id) 
  return state.highlighted
}
```

Reply:
Good point